src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -137,13 +137,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $validAssetDirs = [];
         /** @var BundleInterface $bundle */
         foreach ($kernel->getBundles() as $bundle) {
-            if (!method_exists($bundle, 'getPublicPath')) {
-                @trigger_error('Not defining "getPublicPath()" method is deprecated since Symfony 4.4 and will not be supported in 5.0.', E_USER_DEPRECATED);
-                $publicPath = 'Resources/public';
+            if (!method_exists($bundle, 'getPublicDir')) {
+                @trigger_error('Not defining "getPublicDir()" method is deprecated since Symfony 4.4 and will not be supported in 5.0.', E_USER_DEPRECATED);
+                $publicDir = 'Resources/public';
             } else {
-                $publicPath = $bundle->getPublicPath();
+                $publicDir = ltrim($bundle->getPublicDir(), \DIRECTORY_SEPARATOR);
```

Comment:
should we also trim `/` for cases where the string would be written with `/` even on Windows ?

Reply:
So, can we replace this by the following?

```php
$publicDir = ltrim($bundle->getPublicDir(), '\\/');
```